sql/telemetry/fxa_content_auth_events_v1/view.sql

```diff
@@ -6,6 +6,7 @@ WITH
   SELECT
```

Comment:
both `fxa_content_auth_events_v1/view.sql` and `fxa_content_auth_oauth_events_v1/view.sql` query the wrong table the CTE for auth and content (auth queries content, and content queries auth). it doesn't impact the output, and it's not directly related to this pr, but it's something we should fix.